spec/haskell/src/SEL4/Kernel/Thread.lhs

```diff
@@ -834,18 +834,22 @@ Kernel init will created a initial thread whose tcbPriority is max priority.
 
 > endTimeslice :: Bool -> Kernel ()
 > endTimeslice canTimeoutFault = do
->     ct <- getCurThread
+>     stateAssert cur_tcb'_asrt
```

Comment:
very minor: `stateAssert cur_tcb'_asrt` fits on one line with the string, so you can save a few vertical lines if you want